codec-http2/src/main/java/io/netty/handler/codec/http2/AbstractHttp2StreamChannel.java

```diff
@@ -543,17 +509,11 @@ void fireChildRead(Http2Frame frame) {
             } else {
                 unsafe.notifyReadComplete(allocHandle, true);
```

Comment:
Now that we're essentially eager-reading, why don't we always trigger read complete via `addChannelToReadCompletePendingQueue();`? This should avoid about as many `readComplete` calls as we can.
As an aside, It seems to me that we shouldn't care much what `allocHandle.continueReading()` says at this point, or honestly at all with this strategy.

Reply:
@bryce-anderson good point... let me do this.